src/test/MsQuicTests.h

```diff
@@ -13,7 +13,9 @@
 
 //#define QUIC_COMPARTMENT_TESTS 1
 
-extern QUIC_CREDENTIAL_CONFIG SelfSignedCredConfig;
+extern QUIC_CREDENTIAL_CONFIG ServerCredConfig;
+extern QUIC_CREDENTIAL_CONFIG ServerCredConfigClientAuth;
+extern QUIC_CREDENTIAL_CONFIG ClientCertCredConfig;
```

Comment:
I think we should still have some indication in the name that these can't be validated (which the `SelfSigned` did before).